addons/point_of_sale/models/pos_session.py

```diff
@@ -392,6 +392,9 @@ def _load_account_fiscal_position(self, response, fiscal_positions):
     def _load_account_tax(self, response, taxes):
         response['custom']['account.tax'] = {values['id']: values for values in taxes._convert_to_dict_for_taxes_computation()}
 
+    def _get_available_categories(self):
```

Comment:
i think that this function should be on the `pos.config` model. It's about the config, not the session. Do you agree with this ?

Reply:
Yes I do but I put it here because there was another one in config with the same name

Reply:
can you find a way to unify the 2 functions? It looks like they serve the same purpose